### DISCUSSION TITLE
1.1.3 on Mac

### DISCUSSION THREAD

**Discussion by farmelle** [prb28/vscode-amiga-assembly#194] [General]
Does the 1.1.3 preview have to be specifically built to show up in the extension browser on the Mac?  I'm currently mainly developing on Windows, but the Mac is handy on the sofa in the evening :D

**Comment by prb28** [comment]
No it should be accessible from all os.
Doesn't it appear?
(Don't have my Mac right now)

**Comment by farmelle** [comment]
![image](https://user-images.githubusercontent.com/4208884/148702898-083e1f37-9a0a-4cac-ae7c-84cb4425769d.png)
No switch to preview option.

**Reply by prb28** [reply]
Ohh.. 
do you have the latest version of vscode? It's mandatory for this option.

**Reply by farmelle** [reply]
That was it.  It had downloaded it but not restarted. 🤦‍♂️ Thanks for the help and an excellent plug-in
